metricbeat/module/logstash/test_logstash.py

```diff
@@ -10,6 +10,7 @@
 import urllib.request
 
 
+@metricbeat.parameterized_with_supported_versions
 class Test(metricbeat.BaseTest):
```

Comment:
Maybe we can skip the whole class till tests are fixed, otherwise this is probably going to be starting the containers for nothing in the CI pipelines.

Reply:
yeah, makes sense to me